blob/service.go

```diff
@@ -219,6 +219,13 @@ func (s *Service) getByCommitment(
 	)
 
 	for _, row := range namespacedShares {
+		if len(row.Shares) == 0 {
```

Comment:
shouldn't this only happen in the case that len(namespacedShares) == 1 ?

Reply:
Yes, we expect to get `len(namespacedShares) == 1` from getter